### PR TITLE
add detail page

### PR DESCRIPTION
**Pull request by likuny0415**
1. add support for detail (in the search page, each song will have a detail link and redirect to its detail)
2. has not added comment part
3. met persistent storage problem when refreshing the detail page (data lost when refreshing the page), but when you enter the detail page from the search page, this error will not occur

### CODE REVIEW

**Comment by likuny0415** [comment]
I just deleted node modules from our git, we should not push the node modules to GitHub :)

File: src/omdb/omdb-service.js [hunk 1]
```diff
@@ -1,7 +1,71 @@
...
+        },
+      });
+      localStorage.setItem('token', response.data.access_token);
+      return response.data.access_token;
+    } catch(error) {
+      console.log(error);
+    }
+  }
+
+  export const findAlbumById = async (params) => {
```
Comment (raymondeah) [inline src/omdb/omdb-service.js]: I think this should be 'findSongById', since the search endpoint ends with /tracks and you're using the song id

**Review by raymondeah — COMMENTED** [review]
Looks good overall. Also my bad, should have said this earlier but I think we should move everything to the spotify folder and remove the omdb folder, since we're not using omdb anymore

**Comment by likuny0415** [comment]
I've move code from omdb to spotify, now the omdb folder can be deleted

**Pull request merged**